api/serverless/getRepolinterResult/app.js

```diff
@@ -0,0 +1,67 @@
+const path = require('path');
+const repolinter = require('repolinter');
+const uuidv4 = require('uuid-v4');
+const rimraf = require('rimraf');
+const fs = require('fs');
+const git = require('simple-git')();
+
+async function runRepolinter(gitLink, ruleSet) {
+  const tmpDir = await path.resolve('/tmp', uuidv4());
+  await asyncGitClone(gitLink, tmpDir).catch(error => console.error(error));
+
+  const output = await asyncRunner(tmpDir, ruleSet);
+  rimraf(tmpDir, function() {});
+  return output;
+}
+
+async function asyncRunner(tmpDir, ruleSet) {
+  return new Promise(function(resolve, reject) {
+    const output = repolinter.lint(tmpDir, [], ruleSet);
+    if (output != null) {
+      resolve(output);
+    } else {
+      reject('error');
```

Comment:
In almost all of the cases, repolinter never gives you null response. It will scan an empty folder instead. I have never seen a null response ever. The else statement was added just for unexpected behavior checks.